### PR TITLE
Development cards

### PR DESCRIPTION
**Pull request by wiphadaba**
add some tests and xml files for development cards

### CODE REVIEW

File: app/src/main/java/io/swagslash/settlersofcatan/pieces/Edge.java [hunk 1]
```diff
@@ -96,7 +96,7 @@ public boolean hasRoad() {
      * Determine if the player can build on this edge
      *
      * @param player the player to check for
-     * @return true iff the player has a road to an unoccupied vertex
```
Comment (W3D3) [inline app/src/main/java/io/swagslash/settlersofcatan/pieces/Edge.java]: des hat schon gepasst, iff heißt if and only if 😂 aber is so auch ok 👍

File: app/src/test/java/io/swagslash/settlersofcatan/HexPointPairTests.java [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+    @Test
+    public void equalsTest() throws Exception {
+        HexPointPair hpp1 = new HexPointPair(a, b);
+        HexPointPair hpp2 = (HexPointPair) hpp1;
+
+        // HexPointPair objects which have the same x and y values
+        assertTrue(hpp1.equals(hpp2));
+
+        // not instance of HexPointPair
+        assertFalse(a.equals(b));
```
Comment (W3D3) [inline app/src/test/java/io/swagslash/settlersofcatan/HexPointPairTests.java]: das versteh ich nicht, wieso vergleichst du da HexPoints?

Comment (wiphadaba) [reply app/src/test/java/io/swagslash/settlersofcatan/HexPointPairTests.java]: hab i das falsch verstanden?
/**
     * Checks the two HashPoints for equality, Order does not matter
     *
     * @param o the {@link HexPoint} to which this one is to be checked for equality
     * @return true if the underlying objects of the Pair are both considered
     *         equal
     */

File: app/src/main/java/io/swagslash/settlersofcatan/pieces/Edge.java (hunk 1 above)
Comment (wiphadaba) [reply app/src/main/java/io/swagslash/settlersofcatan/pieces/Edge.java]: ok gut zu wissen xDD

**Pull request merged**